tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_login_bzlite.py

```diff
@@ -0,0 +1,13 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+
+class TestBugzillaLite(GaiaTestCase):
+    def test_login_BugzillaLite(self):
+        self.BugzillaLite = BugzillaLite(self.marionette)
+        slef.BugzillaLite.launch()
+        # enter the username and the password
```

Comment:
Clean up: You can read about the same information in the next line, so I'd remove this comment.